contracts/src/finance/vesting_wallet.rs

```diff
@@ -91,22 +96,21 @@ mod token {
     }
 }
 
-sol_storage! {
-    /// State of the [`VestingWallet`] Contract.
-    pub struct VestingWallet {
-        /// [`Ownable`] contract.
-        Ownable ownable;
-        /// Amount of Ether already released.
-        uint256 _released;
-        /// Amount of ERC-20 tokens already released.
-        mapping(address => uint256) _erc20_released;
-        /// Start timestamp.
-        uint64 _start;
-        /// Vesting duration.
-        uint64 _duration;
-        /// [`SafeErc20`] contract.
-        SafeErc20 safe_erc20;
-    }
+/// State of the [`VestingWallet`] Contract.
+#[storage]
```

Comment:
I've refactored `sol_storage!` -> `#[storage]` everywhere in project. Few advantages it has:
- pure rust code
- body of a struct can be recognized by codecov
- we are in control of `pub` and `pub(crate)` modifiers for struct fields. (`sol_storage!` generates pub by default)